lib/plugins/physics.js

```diff
@@ -12,7 +12,8 @@ module.exports = inject
 const PI = Math.PI
 const PI_2 = Math.PI * 2
 const PHYSICS_INTERVAL_MS = 50
-const PHYSICS_TIMESTEP = PHYSICS_INTERVAL_MS / 1000
+const PHYSICS_TIMESTEP = PHYSICS_INTERVAL_MS / 1000 // 0.05
+const PHYSICS_CATCHUP_TICKS = 4 // limit number of ticks to catchup if loop gets behind
```

Comment:
This seams like a really small number and no way to increase it?

Reply:
This was the number used before, I just added a comment saying what that number is. It is 0.05 seconds, which is 50ms.

Reply:
Oh misread the review. Should I add a configuration option for this?

Reply:
fixed